Anytype/Sources/PresentationLayer/TextEditor/EditorPage/Views/Settings/Icon/Common/Emoji/EmojiGridView.swift

```diff
@@ -63,11 +63,8 @@ struct EmojiGridView: View {
     private func makeEmojiGrid(groups: [EmojiGroup]) -> some View {
         ScrollView(showsIndicators: false) {
             makeGridView(groups: groups)
-        }.gesture(
-            DragGesture().onChanged { _ in
-                UIApplication.shared.hideKeyboard()
-            }
-        )
+        }
+        .scrollDismissesKeyboard(.immediately)
```

Comment:
maybe use interactive? If I want to search and view the results, it would be better if the keyboard doesn't hide

Reply:
Sure, done in 2cc9397adf988dea517ce8c4ebc5893de244d8ec